packages/@okta/vuepress-site/code/library-versions/index.md

```diff
@@ -58,7 +58,7 @@ This table lists the current (and historical) major versions of our open-source
 | [Android OIDC][oidc-android]                          | 1.0.1                              | **Stable**
 | [Angular SDK][sdk-angular]                                      | 0.1.0<br>**1.x**                   | Retired<br>**Stable**                |
 | [ASP.NET SDK][sdk-aspnet]                                      | **1.x**                   | **Stable**                |
-| [ASP.NET Core SDK][sdk-aspnet]                                      | **1.x**                   | **Stable**                |
```

Comment:
Should we keep this and add "Retired" just for future reference (similar to other entries here)?

Reply:
Yeah, good point. I'll add deleted entries back with the corresponding status label.